holoplot/converter.py

```diff
@@ -837,12 +862,12 @@ def contour(self, x=None, y=None, z=None, data=None, filled=False):
         opts = dict(plot=self._plot_opts, style=self._style_opts, norm=self._norm_opts)
 
         # Temporary workaround for bug in contours operation
-        if hv_version < '1.10.5':
-            qmesh = self.image(x, y, z, data)
-        else:
+        if hv_version > '1.10.4':
```

Comment:
You don't want to just require hv=1.10.5?

Reply:
True, put this in before that was released.